Index: isso/setup.py
===================================================================
--- isso.orig/setup.py
+++ isso/setup.py
@@ -43,5 +43,12 @@ setup(
     entry_points={
         'console_scripts':
             ['isso = isso:main'],
-    }
+    },
+    package_data={'isso': [
+        'js/embed.min.js',
+        'js/embed.dev.js',
+        'js/count.min.js',
+        'js/count.dev.js',
+        'defaults.ini',
+    ]},
 )
