--- a/isso/js/build.count.js
+++ b/isso/js/build.count.js
@@ -6,7 +6,7 @@
         "app/text/css": "app/text/dummy"
     },
 
-    name: "components/almond/almond",
+    name: "/usr/lib/nodejs/almond/almond.min.js",
     include: ['count'],
     out: "count.min.js",
     wrap: true
--- a/isso/js/build.embed.js
+++ b/isso/js/build.embed.js
@@ -3,7 +3,7 @@
     mainConfigFile: 'config.js',
     stubModules: ['text', 'jade'],
 
-    name: "components/almond/almond",
+    name: "/usr/lib/nodejs/almond/almond.min.js",
     include: ['embed'],
     out: "embed.min.js",
 
--- a/isso/js/config.js
+++ b/isso/js/config.js
@@ -1,6 +1,7 @@
 var requirejs = {
     paths: {
-        "text": "components/text/text",
+        "fs": "lib/requirejs-jade/fs",
+        "text": "/usr/share/nodejs/requirejs/text/text",
         "jade": "lib/requirejs-jade/jade",
         "libjs-jade": "components/jade/jade",
         "libjs-jade-runtime": "components/jade/runtime"
--- /dev/null
+++ b/isso/js/lib/requirejs-jade/fs.js
@@ -0,0 +1 @@
+define(function(require, exports, module) {});
