=== modified file 'Makefile'
--- old/Makefile	2018-11-11 14:58:15 +0000
+++ new/Makefile	2018-11-11 15:01:44 +0000
@@ -50,7 +50,7 @@
 js: $(ISSO_JS_DST)
 
 man: $(DOCS_RST_SRC)
-	sphinx-build -b man docs/ man/
+	sphinx-build -b man docs/ -d debian/doctrees man/
 	mkdir -p man/man1/ man/man5
 	mv man/isso.1 man/man1/isso.1
 	mv man/isso.conf.5 man/man5/isso.conf.5
@@ -59,7 +59,7 @@
 	$(SASS) --no-cache $(DOCS_CSS_SRC) $@
 
 ${DOCS_HTML_DST}: $(DOCS_RST_SRC) $(DOCS_CSS_DST)
-	sphinx-build -b dirhtml docs/ $@
+	sphinx-build -b dirhtml docs/ -d debian/doctrees $@
 
 site: $(DOCS_HTML_DST)
 

