Origin: commit, https://github.com/jelmer/isso,branch=admin-bool, revision: 704
Author: Jelmer Vernooĳ <jelmer@jelmer.uk>
Last-Update: 2018-11-11
X-Git-Commit: afc90a05cc8cf7209478a35ed4f5da5e70955c20

=== modified file 'MANIFEST.in'
--- old/MANIFEST.in	2017-11-27 21:55:53 +0000
+++ new/MANIFEST.in	2018-11-11 16:20:23 +0000
@@ -11,6 +11,7 @@
 include isso/defaults.ini
 
 include isso/templates/admin.html
+include isso/templates/disabled.html
 include isso/templates/login.html
 include isso/css/admin.css
 include isso/css/isso.css

=== modified file 'docs/docs/configuration/server.rst'
--- old/docs/docs/configuration/server.rst	2018-07-31 06:59:22 +0000
+++ new/docs/docs/configuration/server.rst	2018-11-11 15:33:57 +0000
@@ -355,6 +355,24 @@
 limit
     number of most recent comments to return for a thread
 
+Admin
+-----
+
+Isso has an optional web administration interface that can be used to moderate
+comments. The interface is available under ``/admin`` on your isso URL.
+
+.. code-block:: ini
+
+   [admin]
+   enabled = true
+   password = secret
+
+enabled
+   whether to enable the admin interface
+
+password
+   the plain text pasword to use for logging into the administration interface
+
 Appendum
 --------
 

=== added file 'isso/templates/disabled.html'
--- old/isso/templates/disabled.html	1970-01-01 00:00:00 +0000
+++ new/isso/templates/disabled.html	2018-11-11 15:29:33 +0000
@@ -0,0 +1,28 @@
+<html>
+<head>
+  <title>Isso admin</title>
+  <link type="text/css" href="{{isso_host_script}}/css/isso.css" rel="stylesheet">
+  <link type="text/css" href="{{isso_host_script}}/css/admin.css" rel="stylesheet">
+</head>
+<body>
+  <div class="wrapper">
+    <div class="header">
+      <header>
+        <img class="logo" src="{{isso_host_script}}/img/isso.svg" alt="Wynaut by @veekun"/>
+        <div class="title">
+          <a href="./">
+            <h1>Isso</h1>
+            <h2>Administration</h2>
+          </a>
+        </div>
+      </header>
+    </div>
+    <main>
+      <div id="disabled">
+        Administration is disabled on this instance of isso. Set enabled=true
+        in the admin section of your isso configuration to enable it.
+      </div>
+    </main>
+  </div>
+</body>
+</html>

=== modified file 'isso/views/comments.py'
--- old/isso/views/comments.py	2018-11-03 14:50:55 +0000
+++ new/isso/views/comments.py	2018-11-11 16:20:23 +0000
@@ -1067,8 +1067,10 @@
         )
 
     def login(self, env, req):
+        if not self.isso.conf.getboolean("admin", "enabled"):
+            return render_template('disabled.html')
         data = req.form
-        password = self.isso.conf.get("general", "admin_password")
+        password = self.isso.conf.get("admin", "password")
         if data['password'] and data['password'] == password:
             response = redirect(re.sub(
                 r'/login$',
@@ -1087,6 +1089,8 @@
 
     def admin(self, env, req):
         isso_host_script = self.isso.conf.get("server", "public-endpoint") or local.host
+        if not self.isso.conf.getboolean("admin", "enabled"):
+            return render_template('disabled.html')
         try:
             data = self.isso.unsign(req.cookies.get('admin-session', ''),
                                     max_age=60 * 60 * 24)

=== modified file 'share/isso-dev.conf'
--- old/share/isso-dev.conf	2018-07-31 06:59:22 +0000
+++ new/share/isso-dev.conf	2018-11-11 15:29:33 +0000
@@ -11,7 +11,10 @@
 notify = stdout
 reply-notifications = false
 log-file = /var/log/isso.log
-admin_password = strong_default_password_for_isso_admin
+
+[admin]
+enabled = true
+password = strong_default_password_for_isso_admin
 
 [moderation]
 enabled = false

=== modified file 'share/isso.conf'
--- old/share/isso.conf	2018-07-31 06:59:22 +0000
+++ new/share/isso.conf	2018-11-11 16:20:23 +0000
@@ -59,9 +59,11 @@
 # default url for gravatar. {} is where the hash will be placed
 gravatar-url = https://www.gravatar.com/avatar/{}?d=identicon
 
+[admin]
+enabled = false
+
 # Admin access password
-admin_password = please_choose_a_strong_password
-
+password = please_choose_a_strong_password
 
 [moderation]
 # enable comment moderation queue. This option only affects new comments.

